Semantic HTML: Add `<mark>` and `<time>` tags (#396)
Codecademy/docs: content/html/concepts/semantic-html/terms/time/time.md

---
Title: '<time>'

> Description: 'Represents a time-related piece of information and can take a datetime attribute that makes it machine-readable'

Reviewer: ```suggestion
Description: 'Represents a time-related piece of information and can take a datetime attribute that makes it machine-readable.'
```

> Description: 'Represents a time-related piece of information and can take a datetime attribute that makes it machine-readable'

Reviewer: ```suggestion
Description: 'Represents a time-related piece of information and can take a datetime attribute that makes it machine-readable.'
```

---
Title: '<time>'
Description: 'Represents a time-related piece of information and can take a `datetime` attribute that makes it machine-readable'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'HTML'
  - 'Semantic'
  - 'Syntax'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<time>` semantic tag represents a time-related piece of information, including:
* A specific date in the Gregorian calendar.
* The time on a 24-hour clock.
* A [valid time duration](https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#valid-duration-string). 
The `<time>` element accepts a `datetime` attribute that translates the info to a machine-readable format.

## Syntax

```html
<time>
  <!-- Date and/or time info goes inside the tag -->
</time>
```

The `<time>` element requires opening and closing tags that wrap around a block of text. 

Its `datetime` attribute should be in a valid format for it to be translated to machine-readable info, see below examples.

## Example

> ### Using the <time> element inside of another element

Reviewer 2: ```suggestion
### Using the `<time>` Element Inside of Another Element
```

---
Title: '<time>'

> Description: 'Represents a time-related piece of information and can take a `datetime` attribute that makes it machine-readable'

Reviewer: ```suggestion
Description: 'Represents a time-related piece of information and can take a datetime attribute that makes it machine-readable'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:

>   - 'HTML'
>   - 'Semantic'
>   - 'Syntax'
>   - 'Tags'

Reviewer: ```suggestion
  - 'Tags'
```

CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<time>` semantic tag represents a time-related piece of information, including:

- A specific date in the Gregorian calendar.
- The time on a 24-hour clock.
- A [valid time duration](https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#valid-duration-string).
  The `<time>` element accepts a `datetime` attribute that translates the info to a machine-readable format.

## Syntax

```html
<time>
  <!-- Date and/or time info goes inside the tag -->
</time>
```

The `<time>` element requires opening and closing tags that wrap around a block of text.

Its `datetime` attribute should be in a valid format for it to be translated to machine-readable info, see below examples.

## Example

> ### Using the `<time>` Element Inside of Another Element

Reviewer: Can we turn this heading into regular text since this explains the html snippet below?

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'HTML'
  - 'Semantic'
  - 'Syntax'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<time>` semantic tag represents a specific date (in the Gregorian calendar), time (on a 24-hour clock) or a [valid time duration](https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#valid-duration-string). Its `datetime` attribute translates the info to a machine-readable format.

## Syntax

```html
<time>
  <!-- Date and/or time info goes inside the tag -->
</time>
```

`<time>` requires a closing tag and wraps around a block of text.
Its `datetime` attribute should be a in a valid format for it to be translated to machine-readable info, see below examples.

## Example

### Time string

```html
05:50:30
```

### Week string

```html
1995-W32
```

### Date string

```html
2021-10-24
```

### Duration string

```html
10h 35m 24s
```

### Local date and time string

```html
2021-10-24 18:38
```

### Global date and time string

```html
2021-10-24 18:39-0100
```

### Yearless date string

```html
10-24
```

### Year string

```html
2021
```

### Month string

```html
2021-10

> ```

Author: Hi @Dusch4593, 
Thank you for your suggestions, which I'm implementing. Regarding this last one, do you mean to remove the previous examples and only add the two you mentioned, or add these two to the existing ones?

> ```

Reviewer 2: I'd recommend the HTML examples I suggested only because we're using ```html codeblocks. Plus, it would make for easy copy/pasting for testing/learning purposes.
